### PR TITLE
Feature/22076 waypoint error

### PR DESCRIPTION
**Pull request by nighthawk**

### CODE REVIEW

**Review by MarianoTucat — COMMENTED** [review]
LGTM

File: docs/specs/tripgo.openapi.yaml [hunk 1]
```diff
@@ -5621,6 +5626,21 @@ components:
...
+          description: "A description of the error"
+        errorCode:
+          type: integer
+          description: "Optional error code"
+        usererror:
+          type: boolean
+          description: "if this error should be shown to the user"
+      required:
+        - error
+        - usererror
```
Comment (damianVarela) [inline docs/specs/tripgo.openapi.yaml:5643]: couldn't we just reuse the one that has the error codes? this has the same structure.

**Review by damianVarela — APPROVED** [review]

File: docs/specs/tripgo.openapi.yaml (hunk 1 above)
Comment (nighthawk) [reply docs/specs/tripgo.openapi.yaml:5643]: It does, but all those routing error codes don’t make sense in the more generic errors or reporting them in the docs for endpoints that aren’t routing related.

Comment (damianVarela) [reply docs/specs/tripgo.openapi.yaml:5643]: i see!

**Pull request merged**